app/javascript/onboarding/components/ProfileForm.jsx

```diff
@@ -169,7 +266,6 @@ ProfileForm.propTypes = {
   currentSlideIndex: PropTypes.func.isRequired,
   communityConfig: PropTypes.shape({
     communityName: PropTypes.string.isRequired,
-    communityDescription: PropTypes.string.isRequired,
```

Comment:
Curious why this was removed from the required `propTypes` (but not anything else)?

Reply:
@vaidehijoshi I assume that communityDescription was at some point being used in the Profile Form, but I noticed that there was no reference to it any longer (we only use communityName), hence I removed it.